### PR TITLE
fix: Bedrock support

### PR DESCRIPTION
**Pull request by JGalego**
Fixes https://github.com/VinciGit00/Scrapegraph-ai/issues/226

**Example:** https://gist.github.com/JGalego/67a3db0005f5ed052224e549e8ee627c

image

### CODE REVIEW

File: scrapegraphai/graphs/abstract_graph.py [hunk 1]
```diff
@@ -146,12 +147,12 @@ def _create_llm(self, llm_config: dict, chat=False) -> object:
             except KeyError as exc:
                 raise KeyError("Model not supported") from exc
             return Gemini(llm_params)
-        elif "claude" in llm_params["model"]:
+        elif llm_params["model"].startswith("claude"):
```
Comment (f-aguzzi) [inline scrapegraphai/graphs/abstract_graph.py:150]: Good idea, could be used in other points of this file

**Review by f-aguzzi — CHANGES_REQUESTED** [review]
I'm not a member of the team, however:

- direct contributions to the `main` branch are not accepted, as stated in the [contributing policy](https://github.com/VinciGit00/Scrapegraph-ai/blob/pre/beta/CONTRIBUTING.md). You have to work on the `pre/beta` branch (the embedding problem is already fixed in that branch, by the way)

You might still improve two things on that branch too, though:
- look into the duplicate `Claude` and `Anthropic` classes in the `models` folder (looks like they're doing the same thing)
- swap the community `BedrockEmbeddings` with the official `BedrockEmbeddings`

File: scrapegraphai/graphs/abstract_graph.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
 """
 from abc import ABC, abstractmethod
 from typing import Optional
+from langchain_aws import BedrockEmbeddings
 from langchain_openai import AzureOpenAIEmbeddings, OpenAIEmbeddings
 from langchain_community.embeddings import HuggingFaceHubEmbeddings, OllamaEmbeddings
 from ..helpers import models_tokens
-from ..models import AzureOpenAI, Bedrock, Gemini, Groq, HuggingFace, Ollama, OpenAI, Anthropic
+from ..models import Anthropic, AzureOpenAI, Bedrock, Gemini, Groq, HuggingFace, Ollama, OpenAI, Anthropic
```
[the remark this answers is not in the document]
Comment (JGalego) [reply scrapegraphai/graphs/abstract_graph.py]: IDE didn't wrap around that last import. Good catch!

**Comment by JGalego** [comment]
Superseded by https://github.com/VinciGit00/Scrapegraph-ai/pull/228

**Pull request closed without merging**